api_tests/lib/actor.js

```diff
@@ -4,10 +4,13 @@ const fs = require("fs");
 const bitcoin = require("./bitcoin.js");
 const ethutil = require("ethereumjs-util");
 
-const bitcoin_rpc_client = bitcoin.create_client();
+// TODO: pass value instead of using global config
```

Comment:
:upside_down_face: Would that be so hard to do? :)

Reply:
I just want to draw a line in the sand now so that we can get this one merge and I can then do it straight away.

This is already a `+1,002 −758` PR. The number is likely to triple if I don't stop now.